### PR TITLE
functional tic-tac-toe game

### LINKED ISSUE #2: Present about the use of Event-State Carried Transfer

### PR DESCRIPTION
**Pull request by igakigongo**
**Main Tasks**
- Laid out directory structure as proposed in the README file.
- Add a factory for creating players
- Add some game controllers (use JavsScript Modules for single instance components and hiding of data that does not have to live in the global scope)
- Allow 2 players to play the game (one may be a computer-based player)

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,33 +1,32 @@
...
-            "error",
-            "double"
-        ],
-        "semi": [
-            "error",
-            "always"
-        ]
-    }
-}
+	"env": {
```
Comment (bellom) [inline .eslintrc.json:2]: @igakigongo, one more thing!
You made some changes in this file, check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/tree/master/javascript)
image
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/

File: .eslintrc.json [hunk 2]
```diff
@@ -1,33 +1,32 @@
...
+		"quotes": [
+			"error",
+			"single"
+		],
+		"semi": [
+			"error",
+			"always"
+		]
+	}
+}
```
Comment (shloch) [inline .eslintrc.json]: let the configuration of this file be exactly that that of microverse file
see config here --> https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json

**Review by shloch — CHANGES_REQUESTED** [review]
## perfect execution @igakigongo 

Clean code, logic, organisation and documentation.

Only have one very small thing to conclude with, the linter configuration as stated below.
Let's fix this and move to next project

**Review by bellom — APPROVED** [review]
## Hi @igakigongo  :clap: 

**Great work making all the necessary corrections from the previous review.**

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck in the next project :slightly_smiling_face:_ 

Best,

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by igakigongo** [comment]
Bedankt Sean en tot kijk.

**Pull request merged**